### PR TITLE
[11.0][MIG] w_sale_hide_empty_category

### PR DESCRIPTION
**Pull request by remytms**
Migrate the `website_sale_hide_empty_category` module to v11.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

**Comment by remytms** [comment]
Ok, I will redo it properly following the tutorial. :) Thanks for your comment. :)

File: website_sale_hide_empty_category/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Website Sale - Hide Empty Categories",
+    "summary": "Hide any Product Categories that are empty",
+    "version": "11.0.1.0.0",
+    "category": "Website",
+    "website": "https://laslabs.com/",
```
Comment (nikul-serpentcs) [inline website_sale_hide_empty_category/__manifest__.py]: @remytms Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: website_sale_hide_empty_category/__manifest__.py (hunk 1 above)
Comment (remytms) [reply website_sale_hide_empty_category/__manifest__.py]: @nikul-serpentcs , Thanks for your review ! :) It's now fixed.

**Comment by simahawk** [comment]
wouldn't be better to move this module to OCA/e-commerce ?

**Comment by pedrobaeza** [comment]
Yes, this one is a dangling one that should be moved there.

**Comment by remytms** [comment]
@nikul-serpentcs , Hi, I've done the changes. Could you review this PR again ? :)

Travis check doesn't pass but it's not the fault of this PR.

**Comment by remytms** [comment]
@simahawk , Hi Simone, could you help me by reviewing this PR ? :)

**Comment by remytms** [comment]
@simahawk , @pedrobaeza So, should I open a new PR in OCA/e-commerce with this module  ? :)

**Comment by simahawk** [comment]
@remytms yes please :)
You'll get reviews there.

**Comment by remytms** [comment]
Ok, I will do that. :)

Thanks

**Comment by remytms** [comment]
I have created this PR : https://github.com/OCA/e-commerce/pull/256

**Pull request closed without merging**